src/applications/edu-benefits/0994/components/AddressViewField.jsx

```diff
@@ -11,13 +12,10 @@ export const AddressViewField = ({ formData }) => {
 
   return (
     <p className="blue-bar-block">
-      {street && street}
-      {street && <br />}
-      {street2 && street2}
-      {street2 && <br />}
-      {street3 && street3}
-      {street3 && <br />}
-      {city && city} {state && state} {postalString && postalString}
+      {street && [street, <br />]}
+      {street2 && [street2, <br />]}
+      {street3 && [street3, <br />]}
+      {city} {state} {postalString}
```

Comment:
Refactored the address and removed duplicate checks

Reply:
This change is now causing this warning to show up

![Screen Shot 2020-03-26 at 2 18 38 PM](https://user-images.githubusercontent.com/136959/77687727-df327b00-6f6c-11ea-950c-b0ade88793c7.png)

Reply:
@jhonnyoddball I think we should add the key prop to the br?

Reply:
Oops! I accidentally added the screenshot from form 526 to this PR. It's the same error either way.

I plan on providing a fix for our form using the following JSX:

```js
  {street && (
    <>
      {street}
      <br />
    </>
  )}
  {street2 && (
    <>
      {street2}
      <br />
    </>
  )}
  {street3 && (
    <>
      {street3}
      <br />
    </>
  )}
```

It's not pretty, but it works.